src/main/java/se/sundsvall/partyassets/pr3import/PR3Importer.java

```diff
@@ -113,35 +113,33 @@ Result importFromExcel(final InputStream in) throws IOException {
 
             // Process the rest of the rows
             for (var rowIndex = 1; rowIndex < rows.size(); rowIndex++) {
-                var row = rows.get(rowIndex);
+                final var row = rows.get(rowIndex);
 
                 // Create an asset (create request, to take advantage of validation constraints) and
                 // fill in the static info
-                var assetCreateRequest = new AssetCreateRequest()
+                final var assetCreateRequest = new AssetCreateRequest()
```

Comment:
This could be move to a mapper class (separation of concerns)

Reply:
Out of scope.

Reply:
It's not a major thing to do, just use some of that "boy-scout-rule"-spirit...

Reply:
ahh... you gotta give him that hawk tuah and spit on that thang.

Reply:
Its not a major thing maybe. But its not in the scope of this story. Also this PR3Importer thing could possibily just be deleted all together at this point, but the person that knows is on vacation.

So its out-of-scope and will not be done by me.